packages/cli/test/integration.test.ts

```diff
@@ -133,7 +136,7 @@ async function vcLink(t, projectPath) {
     }
   );
 
-  t.is(exitCode, 0, formatOutput({ stderr, stdout }));
```

Comment:
Will come back through and make all the test failure messages use `formatOutput`